Dockerfile-development

```diff
@@ -0,0 +1,50 @@
+FROM quay.io/3scale/s2i-openresty-centos7:master
+MAINTAINER David Ortiz <dortiz@redhat.com>
```

Comment:
I think we can get rid of the `MAINTAINER` in favour of `LABEL maintainer`

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Reply:
I didn't know it was deprecated. 
Fixed :+1: